### PR TITLE
Update checkout page to match design

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Update checkout page to use new design.

Fix issue passing onClick to next link.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1169** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1169" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1169?utm_source=stack-comment).

File: apps/store/src/components/CountrySelectorPage/CountrySelectorPage.tsx [hunk 1]
```diff
@@ -40,11 +40,10 @@ export const CountrySelectorPage = (props: StoryblokPageProps) => {
             <Link
               key={country}
               href={`/${toRoutingLocale(countryData.defaultLocale)}`}
-              onClick={() => onHandleClick(countryData.defaultLocale)}
               passHref
               legacyBehavior
             >
-              <Button>
+              <Button onClick={() => onHandleClick(countryData.defaultLocale)}>
```
Comment (guilhermespopolin) [inline apps/store/src/components/CountrySelectorPage/CountrySelectorPage.tsx:46]: Nit: not part of this PR but maybe rename `onHandleClick` --> `handleClick`? 🤔

**Review by guilhermespopolin — APPROVED** [review]
👌🏻
